contracts/lib/OrderFulfiller.sol

```diff
@@ -157,10 +157,16 @@ contract OrderFulfiller is
         bytes32 fulfillerConduitKey,
         address recipient
     ) internal {
-        // Derive order duration, time elapsed, and time remaining.
-        uint256 duration = orderParameters.endTime - orderParameters.startTime;
-        uint256 elapsed = block.timestamp - orderParameters.startTime;
-        uint256 remaining = duration - elapsed;
+        uint256 duration;
+        uint256 elapsed;
+        uint256 remaining;
+        // Skip underflow checks as startTime <= block.timestamp < endTime
```

Comment:
to me, it's not immediately obvious that this is true and requires analysing all calling contexts.

Reply:
> to me, it's not immediately obvious that this is true and requires analysing all calling contexts.

Your'e right, but the only call to this function is after verifying these conditions are true

Reply:
It would be nice if the comment was more clear about 1) where this validation happens and 2) mention all the callers and that they do indeed go through this validation step.
Otherwise, when this code is maintained in the future it's easy to do mistakes if another function calls this.